src/main/kotlin/rhmodding/bread/editor/AnimationsTab.kt

```diff
@@ -321,50 +381,78 @@ open class AnimationsTab<F : IDataModel>(editor: Editor<F>) : EditorSubTab<F>(ed
                             }
                         }
                     }
+                    children += Button("Export as PNG sequence").apply {
+                        tooltip = Tooltip("Exports each frame of animation to PNG, including delays.")
```

Comment:
"Exports each frame of the animation as a PNG file. There will be an image generated for every frame, even if the previous frame uses the same step. The current panning and zoom level will be used."